src/html/Engine/syncEventList.js

```diff
@@ -130,9 +200,12 @@ function getBoxUrl(eventCellUrl, token) {
 }
 
 // Get event details
-function getEventInfo(eventCellBoxUrl, token) {
+function getEventInfo(eventCellBoxUrl, token, targetUpdate) {
     var url = eventCellBoxUrl + "OData/Events";
-    var filter = "$filter=startDate%20ge%20datetimeoffset'" + moment.tz("Asia/Tokyo").startOf("day").toISOString() + "'";
+    var filter = "";
+    if (targetUpdate) {
+        filter += "$filter=__updated%20ge%20datetimeoffset'" + moment.tz(targetUpdate, "Asia/Tokyo").toISOString() + "'";
```

Comment:
Easier and bug free to use encodeURIComponent?
Example:
encodeURIComponent("a or b")
"a%20or%20b"